dotcom-rendering/src/model/decideBadge.ts

```diff
@@ -1,6 +1,11 @@
 import type { DCRBadgeType } from '../types/badge';
 import type { Branding } from '../types/branding';
 
+export const BADGE_THIS_IS_EUROPE = {
```

Comment:
```suggestion
const BADGE_THIS_IS_EUROPE = {
```
does it need exporting?